src/dispatch/static/dispatch/src/components/DateWindowInput.vue

```diff
@@ -52,6 +52,15 @@ let today = function () {
 
 const DATE_FORMAT = "yyyy-MM-dd"
 
+function convertStringToDate(value) {
+  let parts = value.split("-")
+  let year = parseInt(parts[0], 10)
+  let month = parseInt(parts[1], 10) - 1
```

Comment:
subtract 1 from the month? So if the month is 4, it will always make it 3? Then the next time it runs, this.modelValue.end will have month 3 so then it will be 2?

Reply:
Stupidly, JavaScript months are 0-based.